src/RuleSet/Sets/SymfonySet.php

```diff
@@ -59,6 +59,11 @@ public function getRules(): array
                     'inheritDocs' => 'inheritDoc',
                 ],
             ],
+            'global_namespace_import' => [
```

Comment:
Hey!
would you mind raising a PR for Symfony repo to apply code changes that this config would introduce? I wonder if the community will approve exact this config after seeing the results, or maybe they would like to adjust it one way or another

Reply:
Hey, do you mean a PR changing the code in ways that triggers this rule to make sure we agree? 
We (the core team) are constantly rejecting the addition of `use` statements for all the elements configured here (classes, functions and constants) so I can tell this is what we want :)